src/app/data/forms/separator.ts

```diff
@@ -1,11 +1,17 @@
-import { Control } from './control';
+import { BaseControl } from './base-control';
 import { XmlClass } from '../../decorators/xml.decorators';
+import { Content } from './content';
 
 @XmlClass('separator')
-export class Separator extends Control {
+export class Separator extends BaseControl implements Content {
```

Comment:
Ich glaube nicht, dass wir da noch ein Interface brauchen.

Reply:
Das Interface war als MarkerInterface gedacht. Ich habe gesehen, dass man das jetzt besser mit Decoratern macht.
Alle Controls, die in ein anderes Control eingefügt werden können (also nicht Form) würden dann diesen Decorater bekommen und damit kann auch kein Form in ein Form eingefügt werden.

Reply:
Eine weitere Alternative wäre ein Flag in der Baseklasse. Das ist wahrscheinlich die einfachste Möglichkeit